pkg/u-boot/Dockerfile

```diff
@@ -38,7 +38,7 @@ ADD ${RASPBERRY_FIRMWARE_BLOBS}/boot/fixup4.dat /tmp/rpi/fixup4.dat
 ADD ${RASPBERRY_FIRMWARE_BLOBS}/boot/start4.elf /tmp/rpi/start4.elf
 # hadolint ignore=SC3060
 RUN  for i in /tmp/rpi/overlays/*.dts ; do                                         \
-          scripts/dtc/dtc -@ -I dts -O dtb -o "${i/.dts/.dtbo}" "$i" && rm "$i"     ;\
```

Comment:
Does this mean we can remove scripts/dtc since you are installing the dtc package?

Reply:
The main reason for using the dtc package is that in the past, we used the dtc from the uboot source that was built when building u-boot. Now we are building overlays before we are building u-boot and scripts/dtc/dtc is not compiled yet.

I would like to keep the logic of the current build, that for each platform the preparation takes place separately and at the end we simple do `make` for u-boot.

Reply:
Or we can move the overlays build command to this block, at this point, scripts/dtc/dtc will already be built:
https://github.com/lf-edge/eve/blob/85ba1cc0d759f635ad005e1d1ff89bb0ad79a820/pkg/u-boot/Dockerfile#L63-L65